src/app/garfile/review/index.njk

```diff
@@ -33,13 +34,13 @@
       {% include "check-your-answers.njk" %}
 
       {% if isJourneyUkInbound%}
-        <form action="/garfile/review" method="post" class="form">
+        <form id="submitGarForm" action="/garfile/review" method="post" class="form">
```

Comment:
I noticed you have done some refactoring.  I don't see **.form** used anywhere or in the CSS files.  I think you can delete this class from all the forms.

Reply:
Sure, I will do so.

Reply:
Changes made.